docs/001_develop/01_development-environment/006_custom-elements-lsp/index.mdx

```diff
@@ -86,44 +91,46 @@ npm i --save-dev @genesiscommunitysuccess/custom-elements-lsp @genesiscommunitys
 	- Open VSCode and navigate to the `tsconfig.json` file.
 	- Copy the `plugins` block shown below.
 
-```
+```json
```

Comment:
I think it makes sense to just provide the configuration option that most people will want. The likelehood that someone with a custom setup in their repo is configuring the plugin is quite small, any legacy projects I will probably end up helping them install it myself if needed